ddtrace/opentelemetry/span.go

```diff
@@ -171,3 +203,115 @@ func toSpecialAttributes(k string, v attribute.Value) (string, interface{}) {
 		return k, v.AsInterface()
 	}
 }
+
+const (
+	httpServerKey    = "http.server.request"
+	httpClientKey    = "http.client.request"
+	operationNameKey = "operation.name"
+)
+
+func remapOperationName(spanKind oteltrace.SpanKind, attrs map[string]interface{}) string {
```

Comment:
```suggestion
func (s *span) createOperationName() string {
```
Nothing is really being "remapped" here. It's being crafted/created based on the attributes of a span. You can also then make this a method on the (*span) type since this algorithm is solely based on the unique span properties anyway, and you don't have to pass in the parameters.

Reply:
True, I couldn't do it before since it was used before span was created, thanks !